python/nn4k/nn4k/executor/huggingface/base/hf_llm_executor.py

```diff
@@ -188,57 +189,87 @@ def load_model(self, args: dict = None, mode=None, **kwargs):
         if self.model_mode == mode and self._model is not None:
             return
 
+        args = args or self._init_args
+
         from transformers import HfArgumentParser
         from nn4k.executor.huggingface import HFModelArgs
 
         parser = HfArgumentParser(HFModelArgs)
+        hf_model_args: HFModelArgs
         hf_model_args, *_ = parser.parse_dict(args, allow_extra_keys=True)
 
         self.model_mode = mode
         self._tokenizer = self._hf_tokenizer_loader(hf_model_args)
+
+        # TODO thinking about resume checkpoint in loading steps
```

Comment:
add a `TODO` issue?

Reply:
This is done and I forgot to delete that TODO, thanks for reminding.